src/Prometheus.Client.HttpClient/HttpClientMetricsOptions.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Prometheus.Client.Collectors;
+
+namespace Prometheus.Client.HttpClient
+{
+    public class HttpClientMetricsOptions
+    {
+        /// <summary>
+        /// Allows you to override the registry for default metric.
+        /// Value will be ignored if a custom metric is set.
```

Comment:
Remove this "Value will be ignored if a custom metric is set.". We don't use custom metric.